src/python/espressomd/interactions.pxd

```diff
@@ -154,6 +157,14 @@ IF GAY_BERNE:
                                  double k1, double k2,
                                  double mu, double nu);
 
+IF THOLE:
+    cdef extern from "thole.hpp":
+        int thole_set_params(int part_type_a, int part_type_b, double scaling_coeff, double q1q2);
+
+cdef extern from "forcecap.hpp":
```

Comment:
This looks like a merge regression to me.

Reply:
True, fixed that.